src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java

```diff
@@ -0,0 +1,44 @@
+package org.fundacionjala.prog101CoffeeMaker.coffeeMaker.testPieces;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class IndicatorLightTest {
+
+    @Test
+    public void turnOn() {
+        IndicatorLight indicatorLight = new IndicatorLight();
+        boolean expected = true;
+        indicatorLight.turnOn();
+        boolean actual = indicatorLight.getLightState();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void turnOff() {
```

Comment:
the PR has to follow our convention. you need to change the name of method

Reply:
Okay thanks for the observation
Already done